desktop/src/main/java/bisq/desktop/common/utils/GridPaneUtil.java

```diff
@@ -13,119 +14,140 @@
+        gridPane.setVgap(verticalGap);
         gridPane.setPadding(gridPadding);
-        if(col1PercentWidth + col2PercentWidth == 100) {
-            setGridPaneTwoColumnsConstraints(gridPane, col1PercentWidth, col2PercentWidth);
-        }
         return gridPane;
     }
 
     /**
-     *  Get icon and text as a label. See example at Dashboard, multiple trade protocols box
+     *  Get icon and text as a label. See example at Dashboard, multiple trade protocols box.
      */
     public static HBox getIconAndText(String labelStyleClass, String text, String imageId) {
         Label label = new Label(text);
         label.getStyleClass().add(labelStyleClass);
         label.setWrapText(true);
         ImageView bulletPoint = ImageUtil.getImageViewById(imageId);
         HBox.setMargin(bulletPoint, new Insets(-3, 0, 0, 4));
-        HBox hBox = new HBox(15, bulletPoint, label);
-        hBox.setAlignment(Pos.CENTER_LEFT);
-        return hBox;
+        HBox horizontalBox = new HBox(15, bulletPoint, label);
+        horizontalBox.setAlignment(Pos.CENTER_LEFT);
+        return horizontalBox;
     }
 
     /**
-     * Column box using the standard style. See example at Dashboard, multiple trade protocols box
+     *  Get info as a Text wrapped on a TextFlow. This allows for correct wrapping of the text.
      */
-    public static void fillColumnStandardStyle(GridPane gridPane,
-                                          int columnIndex,
-                                          Button button,
-                                          String headline,
-                                          String headlineImageId,
-                                          String info) {
-        String groupPaneStyleClass = "bisq-box-1";
-        String headlineLabelStyleClass = "bisq-text-headline-2";
-        String infoLabelStyleClass = "bisq-text-3";
-        String buttonStyleClass = "large-button";
-        fillColumn(gridPane,
-                columnIndex,
-                button,
-                buttonStyleClass,
-                headline,
-                headlineLabelStyleClass,
-                headlineImageId,
-                info,
-                infoLabelStyleClass,
-                groupPaneStyleClass);
+    public static TextFlow getInfoLabel(String info, String infoLabelStyleClass) {
+        Text infoLabelText = new Text(info);
+        infoLabelText.getStyleClass().add(infoLabelStyleClass);
+        return new TextFlow(infoLabelText);
     }
 
     /**
-     * Column box using a custom style. See example at Bisq Easy, best for beginners section
+     *  Get headline label. If no icon is passed, outputs a simple headline.
+     */
+    public static Label getHeadline(String headline,
+                                    String headlineStyleClass,
+                                    String headlineImageId,
+                                    double graphicTextGap) {
+        if (graphicTextGap <= 0d) {
```

Comment:
Why not allow 0? I got 
```
java.lang.IllegalArgumentException: When an image is passed, the graphicsTextGraph must be greater than 0
	at bisq.desktop.common.utils.GridPaneUtil.getHeadline(GridPaneUtil.java:93)
	at bisq.desktop.main.content.bisq_easy.onboarding.BisqEasyOnboardingView.addTopWidgetBox(BisqEasyOnboardingView.java:140)
	```

Reply:
Resolved. Will only throw if the headlineImageId is not empty and the image size is less then or equals to zero.